src/SimpleGraphs/simpleedge.jl

```diff
@@ -32,3 +32,4 @@ SimpleEdge{T}(e::AbstractSimpleEdge) where T <: Integer = SimpleEdge{T}(T(e.src)
 reverse(e::T) where T<:AbstractSimpleEdge = T(dst(e), src(e))
 ==(e1::AbstractSimpleEdge, e2::AbstractSimpleEdge) = (src(e1) == src(e2) && dst(e1) == dst(e2))
 hash(e::AbstractSimpleEdge, h::UInt) = hash(src(e), hash(dst(e), h))
+<(e1::AbstractSimpleEdge, e2::AbstractSimpleEdge) = Tuple(e1) < Tuple(e2)
```

Comment:
Maybe it would be better to define `isless` instead? `<` has `isless` as a fallback anyways.

I would also not relay on `Tuple` here but rather define it in terms of `src` and `dst`. The reason for that is, that we also do it this way for `==` and `hash`. Otherwise it could be, that a custom edge type overrides `Tuple`, which could lead to the weird situation that `e1 < e2` but also `e1 == e2`.

Reply:
```suggestion
isless(e1::AbstractSimpleEdge, e2::AbstractSimpleEdge) = (src(e1) < src(e2)) || ((src(e1) == src(e2)) && (dst(e1) < dst(e2)))
```